### DISCUSSION TITLE
[Wiki] About `$dir` in Pre/Post (un)install scripts

### DISCUSSION THREAD

**Discussion by lewis-yeung** [ScoopInstaller/Scoop#5062] [General]
In [Pre/Post (un)install scripts](https://github.com/ScoopInstaller/Scoop/wiki/Pre-Post-(un)install-scripts), it says that `$dir` will expand to the path of the app's `current` directory. However only in a `post_install` script will it actually expand to the path of `current`, but in the other three and the installer script, it expands to the path of the actual version directory. I think there should be detailed descriptions for this.

**Comment by rashil2000** [comment]
Updated https://github.com/ScoopInstaller/Scoop/wiki/Pre-Post-(un)install-scripts/_compare/2463bf11d7a83acaa9365b85d29a12987d79e5c4
